src/opnsense/mvc/app/controllers/OPNsense/Unbound/forms/general.xml

```diff
@@ -63,6 +63,7 @@
         <type>checkbox</type>
         <help>
             <![CDATA[If this option is set, then DHCP static mappings will be registered in Unbound, so that their name can be resolved.
+            DHCP lease without a specified static ip cannot be register.
```

Comment:
Static mappings are not leases. IP addresses are more or less implied for DNS to resolve but clarifying it in the first sentence wouldn’t hurt I suppose for all those who do read…

Reply:
Does `DHCP static mapping which doesn't contain a static ip will not be registered` sound more accurate ?

Reply:
You want to make it affirmative:

… static mappings with an IP address will be registered…